plugins/woocommerce/client/legacy/js/admin/meta-boxes-product-variation.js

```diff
@@ -1118,25 +1104,15 @@ jQuery( function ( $ ) {
 					woocommerce_admin_meta_boxes_variations.ajax_url,
 					data,
 					function ( response ) {
-						var count = parseInt( response, 10 );
+						let count = parseInt( response, 10 );
+						count = isNaN( count ) ? 0 : count;
 
-						if ( 1 === count ) {
-							window.alert(
-								count +
-									' ' +
-									woocommerce_admin_meta_boxes_variations.i18n_variation_added
-							);
-						} else if ( 0 === count || count > 1 ) {
-							window.alert(
-								count +
-									' ' +
-									woocommerce_admin_meta_boxes_variations.i18n_variations_added
-							);
-						} else {
-							window.alert(
-								woocommerce_admin_meta_boxes_variations.i18n_no_variations_added
-							);
-						}
+						const message =
+							count === 1
+								? woocommerce_admin_meta_boxes_variations.i18n_variation_added
+								: woocommerce_admin_meta_boxes_variations.i18n_variations_added;
+
+						window.alert( message.replace( '%qty%', count ) );
```

Comment:
We don't need to replace the substring `%qty%` when we have only one variation.
Maybe we can do something like this:
```
const message = count === 1 ? i18n_variation_added : ( i18n_variations_added ).replace( '%qty%', count );
window.alert( message );
```

And also modify the `i18n_variation_added` text.

Reply:
Fair point. 😄 Done!